src/routes/dbInit/import.spec.ts

```diff
@@ -0,0 +1,95 @@
+// tslint:disable-next-line
+import fetch from 'jest-fetch-mock';
```

Comment:
Didn't see this. Disregard my comment about the not getting the data from the case API.